src/Processors/Files/Upload.php

```diff
@@ -437,6 +442,8 @@ public function fileUpload() : bool {
 					$convert = new SpreadsheetConverter();
 					$convert->setReader( $fileAction );
 					$convert->setFileName( $storedFile );
+					$convert->setSheetByName( $excelSheetByName );
+					$convert->setSheetById( (int)$excelSheetById );
```

Comment:
Note that this way, SheetById is _always_ an integer

Reply:
indeed!